pdr_backend/lake/etl.py

```diff
@@ -214,10 +218,7 @@ def _calc_bronze_start_end_ts(self):
             else datetime.strptime(self.ppss.lake_ss.st_timestr, "%Y-%m-%d_%H:%M")
         )
 
-        to_values = self._get_max_timestamp_values_from(
-            self.raw_table_names, TableType.TEMP
-        ).values()
-
+        to_values = self._get_max_timestamp_values_from(self.raw_table_names).values()
```

Comment:
the raw data tables are not moved to the prod tables yet during the etl process, so we have to use temp tables

Reply:
The raw data tables move to prod tables during GQLDataFactory.
I have updated the tests to make sure that all flows/tests are following this change.

Please see the diagram I shared.